packages/shared/refAutoReset/index.ts

```diff
@@ -2,14 +2,16 @@ import type { MaybeRefOrGetter, Ref } from 'vue'
 import { customRef, toValue } from 'vue'
 import { tryOnScopeDispose } from '../tryOnScopeDispose'
 
+export type RefAutoResetReturn<T = any> = Ref<T>
```

Comment:
i don't think we need a default as we always infer a `T` earlier

```suggestion
export type RefAutoResetReturnType<T> = Ref<T>
```

Reply:
they should always be specifying `T`, though. as it should be inferred from any usages

lets move this to the main thread though as i wrote more in my reply there